db/migrate/20211005102214_add_searchable_fields_to_proceeding_types.rb

```diff
@@ -0,0 +1,15 @@
+class AddSearchableFieldsToProceedingTypes < ActiveRecord::Migration[6.1]
+  def up
+    add_column :proceeding_types, :additional_search_terms, :string
+    execute 'ALTER TABLE proceeding_types ADD COLUMN textsearchable tsvector'
+    execute 'CREATE INDEX textsearch_idx ON proceeding_types USING GIN (textsearchable)'
+    MatterTypePopulator.call
```

Comment:
having populator calls in the migration files caused us problems on Apply before (when the tables are changed). Are these going to be retrospectively removed?

Reply:
Discussed and agreed that, as the populators need to run on each migration in case changes are made, this will remain for now and a new ticket will be written to implement a data migration gem and move these populators into it.